app/frontend/src/AppRoutes.tsx

```diff
@@ -183,35 +183,31 @@ export default function AppRoutes() {
       {
         // COMMUNITIES
       }
-      {process.env.REACT_APP_IS_COMMUNITIES_ENABLED && (
-        <>
-          <AppRoute isPrivate exact path={communityRoute}>
-            <CommunityPage />
-          </AppRoute>
-          <AppRoute isPrivate path={editCommunityPageRoute}>
-            <EditCommunityInfoPage />
-          </AppRoute>
+      <AppRoute isPrivate exact path={communityRoute}>
+        <CommunityPage />
+      </AppRoute>
+      <AppRoute isPrivate path={editCommunityPageRoute}>
+        <EditCommunityInfoPage />
+      </AppRoute>
 
-          <AppRoute isPrivate path={newPlaceRoute}>
-            <NewPlacePage />
-          </AppRoute>
-          <AppRoute isPrivate path={placeRoute}>
-            <PagePage pageType={PageType.PAGE_TYPE_PLACE} />
-          </AppRoute>
-          <AppRoute isPrivate path={newGuideRoute}>
-            <NewGuidePage />
-          </AppRoute>
-          <AppRoute isPrivate path={guideRoute}>
-            <PagePage pageType={PageType.PAGE_TYPE_GUIDE} />
-          </AppRoute>
-          <AppRoute isPrivate path={discussionRoute}>
-            <DiscussionPage />
-          </AppRoute>
-          <AppRoute isPrivate path={groupRoute}>
-            <GroupPage />
-          </AppRoute>
-        </>
-      )}
+      <AppRoute isPrivate path={newPlaceRoute}>
+        <NewPlacePage />
+      </AppRoute>
+      <AppRoute isPrivate path={placeRoute}>
+        <PagePage pageType={PageType.PAGE_TYPE_PLACE} />
+      </AppRoute>
+      <AppRoute isPrivate path={newGuideRoute}>
+        <NewGuidePage />
+      </AppRoute>
+      <AppRoute isPrivate path={guideRoute}>
+        <PagePage pageType={PageType.PAGE_TYPE_GUIDE} />
+      </AppRoute>
```

Comment:
We might still want to keep these disabled in prod? They still got Aapeli's minimal stuff in there with bad styling...

Reply:
They still got Aapeli's *bad* stuff in there with *minimal* styling...

FTFY

Reply:
There are no links to them though, so does it matter?

Reply:
We probably don't have enough users yet for it to matter that much, but it doesn't seem like a good idea to me to leave half done stuff accessible to the public either...